src/lib/y2partitioner/actions/base.rb

```diff
@@ -42,6 +42,7 @@ def initialize
       def run
         return :back unless run?
 
+        UIState.instance.save_extra_info
```

Comment:
With the changes in my PR, all the buttons and menu options are using `#execute_and_redraw` to run the action:

~~~
execute_and_redraw { action.run }
~~~

Maybe we can move the `#save_extra_info` call to `#execute_and_redraw` method.

Reply:
As discussed at jangouts, I will adapt it in https://github.com/yast/yast-storage-ng/pull/1156 after rebasing these changes.